app/controllers/admin/config/branding_controller.rb

```diff
@@ -38,4 +38,36 @@ def logo
       end
     end
   end
+
+  def fonts
+    previous_default_text_size = SiteSetting.default_text_size
+    SiteSetting::Update.call(
+      guardian:,
+      params: {
+        settings: [
+          { setting_name: "base_font", value: params[:base_font] },
+          { setting_name: "heading_font", value: params[:heading_font] },
+          {
+            setting_name: "default_text_size",
+            value: params[:default_text_size],
+            backfill: params[:update_existing_users].present?,
```

Comment:
```suggestion
            backfill: params[:update_existing_users] == "true",
```

Perhaps we should be more specific here and only accept a truthy value?